.circleci/config.yml

```diff
@@ -102,10 +102,20 @@ jobs:
           environment:
             JUNIT_REPORT_PATH: ./junit/
           when: always
+      -  store_artifacts:
+          path: ~/repo/end-to-end-tests/screenshots
+          destination: /screenshots
+      -  store_artifacts:
+          path: ~/repo/end-to-end-tests/image-compare
+          destination: /image-compare
       - store_test_results:
           path: ~/repo/end-to-end-tests/junit
       - store_artifacts:
           path: ~/repo/end-to-end-tests/junit
+      - store_artifacts:
+          path: ~/repo/end-to-end-tests/imageCompare.html
```

Comment:
i put this out here because it's visible by default in artifacts panel (instead of nested inside folder)